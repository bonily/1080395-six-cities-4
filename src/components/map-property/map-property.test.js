import * as React from "react";
import renderer from "react-test-renderer";
import MapProperty from "./map-property";

const OFFERS = [
  {
    id: 1,
    coords: [52.3909553943508, 4.85309666406198],
  },
  {
    id: 2,
    coords: [52.369553943508, 4.85309666406198],
  },
  {
    id: 3,
    coords: [52.3909553943508, 4.929309666406198],
  },
  {
    id: 4,
    coords: [52.3809553943508, 4.939309666406198],
  },
];

describe(`MapPropertySnapTest`, () => {
  it(`Map should render map on property page component with pins`, () => {
    const tree = renderer
      .create(<MapProperty
        offers = {OFFERS.slice(-3)}
        currentOffer = {OFFERS[0]}
        highlightedPinId = {OFFERS[0].id}
      />,
      {
        createNodeMock: () => document.createElement(`div`)
      }
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
