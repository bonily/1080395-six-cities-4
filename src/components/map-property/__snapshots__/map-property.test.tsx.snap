// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapPropertySnapTest Map should render map on property page component with pins 1`] = `
<div
  style={
    Object {
      "height": 579,
      "marginLeft": "auto",
      "marginRight": "auto",
      "width": 1144,
    }
  }
>
  
</div>
`;
